packages/logistration/package.json

```diff
@@ -52,7 +52,6 @@
     "react-test-renderer": "16.12.0"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^1.9.6 || ^2.0.0",
```

Comment:
[clarification] Why are we removing this peer dependency? `@edx/frontend-platform` is used by the `logistration` package and is up to the MFE (consumer) to install, not this package. This is why `@edx/frontend-platform` is a `devDependency`, not a `dependency`.

It should denote the full range of versions of `@edx/frontend-platform` that `logistration` is compatible with. If one of these existing versions is removed from the existing peer dependency list, this package needs to be released as a breaking change.